bindings/jupyter-modules/modules/jupyter-manifold/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@mlvis/jupyter-manifold",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "webpack --progress",
+    "watch": "webpack --watch --progress"
+  },
+  "dependencies": {
+    "global": "^4.4.0",
+    "react": "^16.8.0",
+    "react-dom": "^16.9.0",
```

Comment:
align react versions (to 16.9 if possible, better hooks support)

Reply:
bumped